### ISSUE TITLE
Netty native build failure

### ISSUE THREAD

**Issue by halimpuckjava** [spring-projects-experimental/spring-native#1388]
hi

i use java 17.0.1, graalvm 21.3.0.r17-grl, spring boot 2.6.2-snapshot, when generate a native executable 

```
java.lang.IllegalArgumentException: Failed to load any of the given libraries: [netty_tcnative_linux_x86_64, netty_tcnative_linux_x86_64_fedora, netty_tcnative_x86_64, netty_tcnative]
```

**Comment by sdeleuze** [comment]
As usual, please provide a repro. Could you also share what OS are you using and if that happens with Naytive Build Tools only, Buildpacks only, or both.

**Comment by sdeleuze** [comment]
We see this error with Spring Boot `2.6.2-SNAPSHOT` so this should be fixed before releasing `0.11.1`. See for example https://ci.spring.io/teams/spring-native/pipelines/spring-native-0.11.x/jobs/build-key-samples-java11/builds/410.

**Comment by sdeleuze** [comment]
Like related to [Netty 4.1.72.Final upgrade](https://github.com/spring-projects/spring-boot/issues/29005).
